File: xorOperation.go
package python

func xorOperation(n int, start int) int {
	nums := make([]int, n, n)
	res := 0
	for i := 0; i < n; i++ {
		nums[i] = start + 2*i
		res = res ^ nums[i]
	}
	return res
}

//runtime 0ms  memory 4.04MB

File: xorOperation.py
class Solution(object):
    def xorOperation(self, n, start):
        """
        :type n: int
        :type start: int
        :rtype: int
        """
        nums =[]
        res =0
        for i in range(0,n):
            nums.append(start + 2*i)
            res = res ^ nums[i]
        return res
#runtime 0ms Memory 12.45MB